exercises/verify.cpp

```diff
@@ -9,10 +9,27 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+bool isThere(int* arr, int val, int N){
+      for(int i = 0; i < N; i++){
+           if(arr[i] == val)
+            return true; 
+      } 
+
+      return false;
+}
+
 int main()
 {
   // placeholder
-  int N[10] = [ 3, 4, 5, 1, 2, 3, 4, 9, 13, 0 ];
+  int dim = 10;
```

Comment:
usa `#define`, eviterai di creare la variabile "dim" in memoria, che occupa abbastanza, in questo caso potresti usare una variabile `short` al posto di `int`, ma l'idea è usare `#define DIM 10` e poi usare `DIM` al posto di `dim`